src/js/components/TaskView.js

```diff
@@ -42,23 +52,13 @@ class TaskView extends mixin(SaveStateMixin) {
   componentWillMount() {
     this.saveState_key = `taskView#${this.props.itemID}`;
     super.componentWillMount();
-
-    MesosStateStore.addChangeListener(
-      EventTypes.MESOS_STATE_REQUEST_ERROR,
-      this.onMesosStateRequestError
-    );
   }
 
-  componentWillUnmount() {
-    super.componentWillUnmount();
-
-    MesosStateStore.removeChangeListener(
-      EventTypes.MESOS_STATE_REQUEST_ERROR,
-      this.onMesosStateRequestError
-    );
+  onStateStoreSuccess() {
+    this.setState({mesosStateErrorCount: 0});
```

Comment:
It would rerender. Supressupdate, just means don't call `forceUpdate` in the storemixin